docs/porting/porting_full_process/integration tests.md

```diff
@@ -0,0 +1,65 @@
+# Integration tests
+
+Mbed OS provides a set of integration tests to help you identify integration issues around storage devices and connectivity devices.
```

Comment:
Yes, this is mainly for people porting, but can also help customers and developers with custom targets and to verify that the platform works well in their environment (in case there are some kind of network limitations).